pages/resources/user-settings-proto.mdx

```diff
@@ -474,13 +474,14 @@ This may also be a snowflake to represent a specific guild in the soundboard sou
 
 ###### Custom Status Structure
 
-| Field         | Type    | Description                                                       |
-| ------------- | ------- | ----------------------------------------------------------------- |
-| text          | string  | The custom status text                                            |
-| emoji_id      | fixed64 | The [ID of a guild's custom emoji](/resources/emoji#emoji-object) |
-| emoji_name    | string  | The unicode character of the emoji                                |
-| expires_at_ms | fixed64 | Timestamp (in milliseconds) of when the custom status expires     |
-| created_at_ms | fixed64 | Timestamp (in milliseconds) of when the custom status was created |
+| Field         | Type    | Description                                                                     |
+| ------------- | ------- | ------------------------------------------------------------------------------- |
+| text          | string  | The custom status text                                                          |
+| emoji_id      | fixed64 | The [ID of a guild's custom emoji](/resources/emoji#emoji-object)               |
+| emoji_name    | string  | The unicode character of the emoji                                              |
+| expires_at_ms | fixed64 | Timestamp (in milliseconds) of when the custom status expires                   |
+| created_at_ms | fixed64 | Timestamp (in milliseconds) of when the custom status was created               |
+| label         | string  | The type of [custom status label](/resources/presence#custom-status-label-type) |
```

Comment:
is this in legacy user settings as well?

Reply:
It is not